mixer/adapter/signalfx/tracing.go

```diff
@@ -160,16 +161,16 @@ func convertInstance(istioSpan *tracespan.Instance) *trace.Span {
 
 	tags := map[string]string{}
 
-	if labels, ok := istioSpan.SpanTags["destination.labels"].(map[string]string); ok {
-		for k, v := range labels {
+	if labels, ok := istioSpan.SpanTags["destination.labels"].(attribute.StringMap); ok {
+		for k, v := range labels.Entries() {
 			if k != "" && v != "" {
 				tags["destination.labels."+k] = v
 			}
 		}
 	}
 
-	if labels, ok := istioSpan.SpanTags["source.labels"].(map[string]string); ok {
-		for k, v := range labels {
+	if labels, ok := istioSpan.SpanTags["source.labels"].(attribute.StringMap); ok {
```

Comment:
note: interesting mistake occurring when attribute values are placed into generic maps in instances.